net/instaweb/rewriter/public/rewrite_options.h

```diff
@@ -4220,6 +4247,8 @@ class RewriteOptions {
   int experiment_percent_;  // Total traffic going through experiments.
   std::vector<ExperimentSpec*> experiment_specs_;
 
+  // Headers to add resource responses.
+  std::vector<NameValue*> resource_headers_;
```

Comment:
re: "consider wrapping this in a CopyOnWrite so that merges are free?"

will do, should I also do so for `custom_fetch_headers_` one line below while I'm at it?